packages/eas-build-job/src/android.ts

```diff
@@ -84,6 +85,7 @@ export interface Job {
     environmentSecrets?: Env;
   };
   builderEnvironment?: BuilderEnvironment;
+  buildResourceClass: BuildResourceClass;
```

Comment:
I'm not sure if this should be a part of the job object, I think it should be treated more like build priority

Reply:
Actually I think I might understand what you mean, intuitively speaking. I'll have to familiarize myself more with build priority and see if it makes sense to think about Resource Class in the same way (with respect to a Job).

Reply:
build priority similarly as resource classes are not needed on the worker, it's a value used to start vm/pod but when building it's not necessary. On the other hand, this is a bit different case because the value is user-defined, so it's a bit different case.

We already keep the image field in the job object, so maybe it makes sense to keep priority here

Reply:
ok nevermind, I forgot that there are 2 enums one with names default/large/m1 and one with specific configurations that build will run on https://github.com/expo/universe/blob/main/server/www/src/data/entities/turtlebuild/TurtleBuildEntity.ts#L100

I agree with Domink's comment here https://github.com/expo/eas-build/pull/113#issuecomment-1144601878 no changes should be necessary in the turtle-v2 and in eas-build repos, enum you are adding here should not be passed to turtle-api